Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -699,6 +700,38 @@ void RenderGrid::computeIntrinsicLogicalWidths(LayoutUnit& minLogicalWidth, Layo
 
     computeTrackSizesForIndefiniteSize(algorithm, GridTrackSizingDirection::ForColumns, gridLayoutState, &minLogicalWidth, &maxLogicalWidth);
 
+    if (isMasonry(GridTrackSizingDirection::ForColumns)) {
+        // The track sizing algorithm will only be run once in this case, since track sizing will not run in the masonry direction.
+        computeTrackSizesForIndefiniteSize(algorithm, GridTrackSizingDirection::ForRows, gridLayoutState, &minLogicalWidth, &maxLogicalWidth);
+
+        auto gridAxisTracksCountBeforeAutoPlacement = currentGrid().numTracks(GridTrackSizingDirection::ForRows);
+        LayoutUnit availableLogicalWidth;
```

Comment:
Since we don't actually need an `availableLogicalWidth` for the combination of arguments we are passing to `computeIntrinsicLogicalWidthUsing` can you just replace this with a `LayoutUnit()` at the call site?. 

It would be nice if that logic was extracted out into some other method that we could use directly but I don't see one readily available :/